lib/inferno/dsl/fhir_evaluator/config/default.yml

```diff
@@ -0,0 +1,9 @@
+Environment:
```

Comment:
Originally config yml was assumed to include list of rules and their individual configurations, which brings possibilities of large ymls with customizations depending on use cases of the evaluator. We don't have clear picture of what it will be. Dylan's suggestion is to make it simplest at this phase. I agree with you that it doesn't have much value as it is.